lib/cli/generators/REACT_NATIVE/index.js

```diff
@@ -44,20 +51,20 @@ export default async (npmOptions, installServer) => {
   packageJson.dependencies = packageJson.dependencies || {};
   packageJson.devDependencies = packageJson.devDependencies || {};
 
+  const dependencies = [
+    `@storybook/react-native@${storybookVersion}`,
+    `@storybook/addon-actions@${actionsVersion}`,
+    `@storybook/addon-links@${linksVersion}`,
+    `@storybook/addons@${addonsVersion}`,
+  ];
+
   if (installServer) {
-    packageJson.devDependencies['@storybook/react-native-server'] = storybookVersion;
+    dependencies.push(`@storybook/react-native-server@${storybookVersion}`);
```

Comment:
This makes it look like we're going to add them to dependencies, rather then devDependencies, WDYT?

Reply:
yea that makes sense, will make it more precise 👍